js/charts/combo.js

```diff
@@ -800,63 +800,61 @@ function combo() {
                 .on('mousemove', _handleMouseMoveFn.call(chart, tooltip, _local_svg))
                 .on('mouseout', _handleMouseOutFn.call(chart, tooltip, _local_svg))
                 .on('click', function (d) {
-                    if (COMMON.COMPARABLE_DATA_TYPES.indexOf(_dimensionType[0]) === -1) {
-                        if (!_print) {
-                            if (broadcast != undefined && broadcast.isThresholdAlert) {
-                                var ThresholdViz = {};
-                                ThresholdViz.ID = parentContainer.attr('vizID');
-                                ThresholdViz.measure = d.tag;
-                                ThresholdViz.measureValue = d.data[d.tag];
-                                ThresholdViz.dimension = _dimension[0];
-                                ThresholdViz.dimensionValue = d.data[_dimension[0]];
-                                broadcast.ThresholdViz = ThresholdViz;
-                                broadcast.$broadcast('FlairBi:threshold-dialog');
-                            }
-                            else {
-                                filter = false;
-                                var confirm = parentContainer.select('.confirm')
-                                    .style('visibility', 'visible');
+                    if (!_print) {
+                        if (broadcast != undefined && broadcast.isThresholdAlert) {
+                            var ThresholdViz = {};
+                            ThresholdViz.ID = parentContainer.attr('vizID');
```

Comment:
these lines are dupilcate set of code for all viz...pls keep them in seperate function